src/pages/academy/videos/_all-videos.tsx

```diff
@@ -40,7 +67,8 @@ const AllVideos = ({ video_data }: AllVideosProps) => {
     const closeVideo = (): void => {
         setShow(false)
         setPlayVideoId('')
-        setTitleParams(undefined)
+        setTitleParams('')
+        queryParams.delete(['t'])
```

Comment:
Can we pass `t` as a string here because your helper accepts both  string and array anyways

Reply:
okay